packages/ui5-library-writer/templates/optional/typescript/src/_.babelrc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "ignore": [
+        "**/*.d.ts"
+    ],
+    "presets": [
+        "transform-ui5",
+        "@babel/preset-typescript"
+    ]
+}
```

Comment:
Should probably be aligned with https://github.com/SAP/open-ux-tools/blob/main/packages/ui5-application-writer/templates/optional/typescript/.babelrc.json#L10

Reply:
Updated